src/js/components/SelectMultiple/stories/Variant.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+
+import { Box, SelectMultiple, Text } from 'grommet';
+
+const dummyOptions = [
+  'Apple',
+  'Orange',
+  'Banana',
+  'Grape',
+  'Melon',
+  'Strawberry',
+  'Kiwi',
+  'Mango',
+  'Raspberry',
+  'Rhubarb',
+];
+
+export const Variant = () => {
+  const [options, setOptions] = useState(dummyOptions.sort());
+  const [valueMultiple, setValueMultiple] = useState([]);
+  const [open, setOpen] = useState(false);
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Box fill align="center" pad="large" gap="large">
+      <Text>SelectMultiple Variant</Text>
+      <SelectMultiple
+        showSelectedInline
+        placeholder="Select"
+        options={options}
+        onSearch={(text) => {
+          // The line below escapes regular expression special characters:
+          // [ \ ^ $ . | ? * + ( )
+          const escapedText = text.replace(/[-\\^$*+?.()|[\]{}]/g, '\\$&');
+
+          // Create the regular expression with modified value which
+          // handles escaping special characters. Without escaping special
+          // characters, errors will appear in the console
+          const exp = new RegExp(escapedText, 'i');
+
+          let next = [...valueMultiple];
+          next = next.filter((o) => exp.test(o));
+          let sortedOptions = dummyOptions.filter((i) => !next.includes(i));
+
+          sortedOptions = sortedOptions.filter((o) => exp.test(o));
+
+          const sortedAllOptions = next.concat(sortedOptions);
+          setOptions(sortedAllOptions);
+        }}
+        value={valueMultiple}
+        open={open}
+        onOpen={() => {
+          // setValueMultiple(intermediateValue);
+          setOpen(true);
+          let next = [...valueMultiple];
+          next.sort();
```

Comment:
Why do we need to `sort()` when opening the drop down?

Reply:
No longer using this code since sort prop was added in latest commit